acceptance/setup/common/005_SyncTime.rb

```diff
@@ -4,4 +4,8 @@
 # Note: QENG-3641 would allow beaker to take care of this for us
 test_name 'Ensure hosts have synchronized clocks'
 
-timesync(hosts, {:logger => logger})
+# QENG-3786 - OSX is already running ntpd and manually running ntpdate will return 1
+applicable_hosts = hosts.select{|host| host['platform'] !~ /osx/}
+if applicable_hosts then
```

Comment:
In this case, it'll just log that the array was empty:  https://github.com/puppetlabs/beaker/blob/master/lib/beaker/shared/host_manager.rb#L107-L113

So yes, if you want to guard against that, check for !Array#empty? but it doesn't seem that serious to just have the log message occur.